cmd/deploy.go

```diff
@@ -137,18 +129,46 @@ This requires an Inertia daemon to be active on your remote - do this by running
 		if err != nil {
 			log.Fatal(err)
 		}
-		buildType, err := cmd.Flags().GetString("type")
+		sendConfig, err := cmd.Flags().GetBool("update-config")
 		if err != nil {
 			log.Fatal(err)
 		}
 
-		// TODO: support other remotes
-		url, err := local.GetRepoRemote("origin")
-		if err != nil {
-			log.Fatal(err)
+		// Check if remote already has repository - if not, this is a first-time
+		// setup and the configuration should be sent to remote
+		if !sendConfig {
+			var resp *http.Response
+			if resp, err = deployment.Status(); err != nil {
+				log.Fatal(err)
+			}
+			status := &common.DeploymentStatus{}
+			if err = json.NewDecoder(resp.Body).Decode(status); err != nil {
+				log.Fatal(err)
+			}
+			if status.CommitHash == "" {
+				sendConfig = true
+			}
+		}
+
+		if sendConfig {
+			println("reading project configuration...")
+			f, err := os.Open(projectConfigFilePath)
+			if err != nil {
+				log.Fatal(err.Error())
+			}
+
+			println("sending configuration to remote...")
+			projectPath := "$HOME/inertia/project"
```

Comment:
No love for Windows VPS 🙁

Reply:
Definitely needs windows tests - I've been hoping to set up Appveyor, but needs @bfbachmann to install it to the Launch Pad account (I can't seem to do it)